build/rollup.config.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path')
+const fs = require('fs')
+const babel = require('rollup-plugin-babel')
+const resolve = require('rollup-plugin-node-resolve')
+const BUNDLE = process.env.BUNDLE === 'true'
+const year = new Date().getFullYear()
+
+const dataPkg = fs.readFileSync(path.resolve(__dirname, '../package.json'))
+const pkg = JSON.parse(dataPkg)
+
+var fileDest = 'bootstrap.js'
+var external = ['jquery', 'popper.js']
+const plugins = [
+  babel({
+    exclude: 'node_modules/**' // only transpile our source code
+  })
+]
+
+if (BUNDLE) {
+  fileDest = 'bootstrap.bundle.js'
+  // remove last entry in external array ton bundle Popper
+  external.pop()
+  plugins.push(resolve())
+}
+
+module.exports = {
+  entry: path.resolve(__dirname, '../js/src/index.js'),
+  format: 'umd',
+  moduleName: 'bootstrap',
+  dest: path.resolve(__dirname, `../dist/js/${fileDest}`),
+  external: external,
+  globals: {
+    jquery: '$',
+    'popper.js': 'Popper'
```

Comment:
If someone want to use Bootstrap in a UMD module a rollupjs config like this would work perfectly without : `window.Popper = Popper` : 

![image](https://user-images.githubusercontent.com/1689750/29466705-2e8f0d02-843e-11e7-8134-274c0d2f29cc.png)